kalite/testing/behave_helpers.py

```diff
@@ -239,9 +243,33 @@ def build_url(context, url):
     return urljoin(context.config.server_url, url)
 
 def go_to_homepage(context):
+    """
+    Goes to the homepage. Raises a TimeoutException if page does not load.
+    """
     url = reverse("homepage")
-    context.browser.get(build_url(context, url))
+    _go_to_url(context, build_url(context, url))
+
+def go_to_coachreports(context):
+    """
+    Goes to coach reports. Raises a TimeoutException if page does not load.
+    """
+    url = reverse("coach_reports")
+    _go_to_url(context, build_url(context, url))
 
+def _go_to_url(context, url, wait_time=MAX_PAGE_LOAD_TIME):
+    """
+    Tries to browse to a page. Waits for the page to load, since WebDriver.get does not.
```

Comment:
I guess my assumption about the test failures in #3809 was wrong, unless there's a race condition because stuff is happening after onload but before the rest of the JS stuff is ready.